sway-fmt-v2/src/items/item_use.rs

```diff
@@ -4,5 +4,7 @@ use sway_parse::ItemUse;
 impl Format for ItemUse {
     fn format(&self, _formatter: &Formatter) -> FormattedCode {
         todo!()
+        // reorder use statements alphabeticaly
```

Comment:
Maybe we can remove this diff? (assuming it is not related to this PR)

Reply:
Ah, you're right! Thank you